kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/utils/TestUtils.java

```diff
@@ -104,4 +112,50 @@ public static String getJsonFileContent(String fileName) {
             throw new UncheckedIOException(e);
         }
     }
+
+    // TO BE REMOVED when introduced in ImageUtils from TestFrame project
```

Comment:
Non blocking: DockerImageName from Testcontainers is has a much nicer API and. is more functionally complete than the class from TestFrame.

Reply:
Maybe we can add this functionality of test containers to the test frame as well

Reply:
Yeah, that's probably what I would do.  Testframe's API fluent style API is much nicer.  static methods are to be avoided.

Reply:
The change maintaining the way it is coded in Test frame is done and merged, but to remove the code added in this PR we need to wait until a new version of Test Frame is released